### DISCUSSION TITLE
Logout issues with Gihub Enterprise  auth enabled

### DISCUSSION THREAD

**Discussion by srthok** [apache/airflow#24561] [Q&A]
**Apache Airflow version**:  1.10.9

**Environment**:

- **Cloud provider or hardware configuration**: AWS
- **OS** (e.g. from /etc/os-release):  Ubuntu 18.04.4 LTS
- **Kernel** (e.g. `uname -a`): `Linux ip-x-x-x-x 5.4.0-1025-aws #25~18.04.1-Ubuntu SMP x86_64 GNU/Linux`
- **Install tools**:  Flask-OAuthlib (0.9.6)
- **Others**:

**What happened**:
    After configuring GHE auth, Login is working as expected and when I hit logout, It says as below and still can see DAGs and able to run DAGs

     
![image](https://user-images.githubusercontent.com/11413448/95154768-3608e080-0758-11eb-8b7d-ffc74e9437a1.png)

  **What you expected to happen**:

     It should invalidate the session and take you  back to the login page

**How to reproduce it**:

Just by enabling GHE auth in airflow..

  Here is what my config looks like..
<
[webserver]
authenticate = True
auth_backend = airflow.contrib.auth.backends.github_enterprise_auth
[github_enterprise]
host = github.xxxxx.yyyyy
client_id = xxxxxxxx
client_secret = xxxxxxx
oauth_callback_route = /admin/ghe_oauth/callback
allowed_teams = 32,947,948,682
>
             
**Anything else we need to know**:

It occurs all the time. I just followed the documentation to configure..http://airflow.apache.org/docs/stable/security.html#oauth-authentication

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by mik-laj** [comment]
Non-RBAC is no longer supported and we plan to drop support for it in Airflow 2.0. Non-RBAC UI now only gets security fixes. I'm afraid no contributor is interested in fixing other bugs and adding new features. I recommend migrating to RBAC and then configuring OAuth2 again. 
See: 
https://github.com/apache/airflow/blob/master/UPDATING.md#drop-legacy-ui-in-favor-of-fab-rbac-ui
https://airflow.readthedocs.io/en/latest/security/access-control.html
https://flask-appbuilder.readthedocs.io/en/latest/security.html

**Comment by srthok** [comment]
Thanks for the info..I enabled rbac..I need to enable Oauth( GHE) and really appreciate if you can point me to  any related documentation to enable GHE auth or sample confg file.Thanks

**Comment by jykae** [comment]
@srthok check out https://issues.apache.org/jira/browse/AIRFLOW-4470

I was able to configure GHE with RBAC based on Cooper Gillan's [comment](https://issues.apache.org/jira/browse/AIRFLOW-4470?focusedCommentId=17016268&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17016268)

@mik-laj I hope this will go to documentation at some point ☝️

**Comment by eladkal** [comment]
@jykae would you be interested in updating the documentation?

**Comment by jykae** [comment]
> @jykae would you be interested in updating the documentation?

@eladkal Sorry, can't remember enough to write docs about this, and would it work with current version 🤷‍♂️ 

Switched to use local DB auth, as we had this issue also with GHE auth #12782
Though we are still looking for centralized authentication solution for our installation.
